src/main/java/org/broadinstitute/hellbender/tools/exome/GetBayesianHetCoverage.java

```diff
@@ -162,7 +162,7 @@
             shortName = HET_CALLING_STRINGENCY_SHORT_NAME,
             optional = true
     )
-    protected double hetCallingStringency = 5;
+    protected double hetCallingStringency = 30;
```

Comment:
I agree with this -- this is something that I wanted to change for a while, thank you!